source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,66 +1,74 @@
 .. _mixer:
 
-Use the mixer tool
-##################
+Mixer tool
+##########
 
-*Mixing* refers to composing an operating system for specific use cases.
-While the default |CL-ATTR| provides options to install bundles for various
-server capabilities, some developers may wish to either augment the
-operating system itself with functionality from their own packages or modify
-the structure of current bundles to cater to their particular needs.
+The mixer tool is used to create a customized image based on |CL-ATTR|. This is
```

Comment:
Instead of "customized image", I would describe it as a "custom OS". Referring to mixer's output as an image is partially true, but only when an image is explicitly generated after the bundle and update content are created.